### PR TITLE
Feature/sessions

### PR DESCRIPTION
**Pull request by danielquinn**
As per the pentest, we've switched from using static tokens to dynamically assigned sessions.

Tagging @adammalinowski and @marcofucci on this one for QA.

### CODE REVIEW

File: wins/tests/test_views.py [hunk 1]
```diff
@@ -396,22 +386,16 @@ def test_advisors_post_fail_no_auth(self):
 
     @override_settings(UI_SECRET=AliceClient.SECRET)
     def _test_post_fail_bad_auth(self, url, data):
-        auth = {"HTTP_AUTHORIZATION": "Token bad-auth"}
-        response = self.alice_client.post(url, data, **auth)
-        self.assertEqual(response.status_code, 401)
+        self.alice_client.login(username="not-a-user", password="fail")
```
Comment (adammalinowski) [inline wins/tests/test_views.py]: Be good to also have a test for a valid username but invalid password for completeness

File: wins/tests/test_views.py [hunk 2]
```diff
@@ -179,14 +175,14 @@ def test_advisor_schema_fail_bad_client(self):
 
     @override_settings(UI_SECRET=AliceClient.SECRET)
     def test_get_notification_list_not_allowed(self):
-        auth = {"HTTP_AUTHORIZATION": "Token {}".format(self.user_token)}
-        response = self.alice_client.get(self.notifications_list, **auth)
+        self.alice_client.login(username=self.user.email, password="asdf")
```
Comment (adammalinowski) [inline wins/tests/test_views.py]: Nitpicking the duplication I introduced, but I'd be nice to DRY this to a `self._login()` helper.

Comment (danielquinn) [inline wins/tests/test_views.py]: Ooh good point.  I can do that, sure.

File: wins/tests/test_views.py (hunk 1 above)
Comment (danielquinn) [inline wins/tests/test_views.py]: Sure ok.  No harm in that.

File: data/urls.py [hunk 3]
```diff
@@ -17,7 +17,12 @@
 router.register(r"notifications", NotificationViewSet)
 
 urlpatterns = [
+
     url(r"^", include(router.urls, namespace="drf")),
-    url(r'^login/', LoggingObtainAuthToken.as_view()),
+
+    # Override DRF's default 'cause ours skips CSRF
```
Comment (danielquinn) [inline data/urls.py]: Fix this.

**Pull request merged**